README.md

```diff
@@ -2580,6 +2580,20 @@ There are spots in the templates where the `T` operator is used.
 
 #### Coming Soon!
 
+### Logging
+
+Form Flow adds the following attributes to the [Mapped Diagnostic Context](https://logback.qos.ch/manual/mdc.html):
+
+1. xForwardedFor
```

Comment:
Thinking about it further, we should probably add a brief description for each of these as others may not know what xForwardedFor is (like I didn't), etc.